README.md

```diff
@@ -304,7 +304,6 @@ for automation including CI/CD pipelines for each application.
 Workflows are located in `.github/workflows` which currently contains the following:
 
 - `ci-orchestrator.yml` - Markdown check and YAML validation, detects changes to start relevant workflows and branch policy status check.
-- `clean-up-cache.yml` - Cleanup GitHub workflow caches for closed branches.
```

Comment:
suggestion: keep the workflow as it still appears to be used. 

https://github.com/Energinet-DataHub/greenforce-frontend/actions/workflows/clean-up-cache.yml

Reply:
If I search across the entire Energinet-Datahub organization, no-one is using it: 

https://github.com/search?q=org%3AEnerginet-DataHub+clean-up-cache&type=code

The [gh-actions-cache](https://github.com/actions/gh-actions-cache) is also deprecated and the repository was archived in October 2024, as the extension has been baked into Github CLI: https://cli.github.com/manual/gh_cache